lib/postgrex.ex

```diff
@@ -147,22 +165,41 @@ defmodule Postgrex do
 
   ## Handling failover
 
-  Some services, such as AWS Aurora, support failovers. This means the
-  database you are currently connected to may suddenly become read-only,
-  and an attempt to do any write operation, such as INSERT/UPDATE/DELETE
-  will lead to errors such as:
-
-      11:11:03.089 [error] Postgrex.Protocol (#PID<0.189.0>) disconnected:
-      ** (Postgrex.Error) ERROR 25006 (read_only_sql_transaction)
-      cannot execute INSERT in a read-only transaction
-
-  Luckily, you can instruct `Postgrex` to disconnect in such cases by
-  using the following configuration:
-
-      disconnect_on_error_codes: [:read_only_sql_transaction]
```

Comment:
Just to confirm: we no longer need this configuration? Is it guaranteed that, if a primary becomes a secondary, our connection will be booted by the server? Or should we add this code to make sure we reconnect when running update queries on read-only mode?

Reply:
I'm not an expert on the topic sadly. The answer might also depend on the replication method used and the implementation. If I'm not mistaken however: 
- If it was an automatic failover, it means the primary instance crashed and/or needed to restart
- If it was a manually triggered failover, the primary instance has to be stopped and restarted at some point before it can be turned into a secondary instance

So yes, either way, every connection would be closed by the previous primary.